typings/index.d.ts

```diff
@@ -1610,10 +1605,8 @@ declare module 'discord.js' {
   export class VoiceChannel extends GuildChannel {
     constructor(guild: Guild, data?: object);
     public bitrate: number;
-    public readonly editable: boolean;
```

Comment:
Note: At the moment, your PR **does not** remove `editable` from src/structures/VoiceChannel.js.